### DISCUSSION TITLE
❓❓ Como posso entrar na organização. ❓❓

### DISCUSSION THREAD

**Discussion by Gabriel-Sousa-Amorim** [UNIVESP-Universidade-Virtual-de-SP/.github#1] [Geral]
# Passo a Passo

1. Se já tiver conta pule para o passo 2, Crie uma conta no Github. [Crie Aqui](https://github.com/signup):
    
    - Precisará de um:
        - E-mail *(Preferível pessoal)*.
        - Nome de usuário.
        - Senha. *(A senha pode ser como você quiser, mas tem de ter 8 caracteres com pelo menos uma letra maiuscula e um número,15 Caracteres sem números e minusculos)*.

![Cadastro Github](https://github.com/UNIVESP-Universidade-Virtual-de-SP/.github/assets/97068125/c5068fd4-562b-430f-9b55-6b7e161a82a4)

2. Após isso venha para a organização da [UNIVESP](https://github.com/UNIVESP-Universidade-Virtual-de-SP) e siga-nos para entrar nela, ou mande em um dos grupos da faculdade o nome de usuário e pedindo explicitamente para te adicionar.

3. Em algum momento sua caixa de notificações **(localizado no canto superior direito no PC, No Android possui uma seção na parte inferior escrita Notificações)** estará com uma bolinha azul sinalizando uma nova notificação.

![Notificações](https://github.com/UNIVESP-Universidade-Virtual-de-SP/.github/assets/97068125/d07a8b0f-0022-442a-8c92-56dac65e712b)

4. Aceite o convite que estará nas notificações e pronto você entrou na organização 🎉🎉🎉.

**Comment by naufragodaweb** [comment]
Olá Gabriel, bom dia!

Meu nome é Renato, sou aluno do 3º bimestre do Eixo de Computação da UNIVESP (Curso de Engenharia da Computação) e gostaria de participar da organização do GitHub da UNIVESP. O motivo é que eu gostaria de replicar aqui, em um fork, o curso de Engenharia da Computação, com todas as disciplinas atuais e as que já saíram da ementa do curso. Uma espécie de nova biblioteca dos alunos, mas com mais controle e nos moldes dos cursos da [OSSU](https://github.com/ossu/computer-science-br), para que não tenhamos mais que passar por impropérios, como os causados pelo fechamento da antiga biblioteca dos alunos ou pela enésima troca do AVA, que dessa vez causou muito prejuízo aos alunos que ainda estão no curso de Engenharia da Computação e perderam todo o acesso às matérias antigas. Tem mais alunos querendo construir esse curso e, se me permitir, informarei a existência deste perfil e solicitarei para eles que me ajudem nessa tarefa. Conto com a sua ajuda também, se possível! Abraços aqui do seu novo colega! :D
